### PR TITLE
schemas: add revision_id to records envelope

### PR DESCRIPTION
**Pull request by pazembrz**
Required for https://github.com/inspirehep/inspirehep/issues/1962

### CODE REVIEW

File: backend/inspirehep/records/marshmallow/base.py [hunk 1]
```diff
@@ -38,6 +38,7 @@ class EnvelopeSchema(RecordSchemaJSONV1):
     """
 
     uuid = fields.String(dump_only=True, attribute="pid.object_uuid")
+    revision_id = fields.Integer(dump_only=True, attribute="metadata.revision_id")
```
Comment (drjova) [inline backend/inspirehep/records/marshmallow/base.py:41]: could you please also update the docstring?

**Review by drjova — APPROVED** [review]
just the comment 👍

File: backend/inspirehep/records/marshmallow/base.py (hunk 1 above)
Comment (pazembrz) [reply backend/inspirehep/records/marshmallow/base.py:41]: :+1:

Comment (pazembrz) [reply backend/inspirehep/records/marshmallow/base.py:41]: Wait. From what I can see docstring is about what Invenio is returning not for what we are returning here... I'm confused.

Comment (drjova) [reply backend/inspirehep/records/marshmallow/base.py:41]: we're extending `RecordSchema` from invenio

**Pull request merged**